libraries/AP_NavEKF3/AP_NavEKF3_Control.cpp

```diff
@@ -247,17 +247,17 @@ void NavEKF3_core::setAidingMode()
             bool rngBcnUsed = (imuSampleTime_ms - lastRngBcnPassTime_ms <= minTestTime_ms);
 
             // Check if GPS is being used
-            bool gpsPosUsed = (imuSampleTime_ms - lastPosPassTime_ms <= minTestTime_ms);
```

Comment:
This looks like a minor regression ; gpsPosUsed was a better name :-)

Reply:
Position  data can now come from sources other than GPS. The comment above is what should change.